$OpenBSD: patch-src_manage_c,v 1.1 2014/02/26 20:07:18 jca Exp $
Erroneous change introduced in 1.4.6.
Applied-Upstream: 5201477cbcb276bd13ec07e3d57b8c6cefe8201a
--- src/manage.c.orig	Thu Feb 20 02:41:38 2014
+++ src/manage.c	Thu Feb 20 02:41:13 2014
@@ -233,7 +233,7 @@ get_wmname (Window w)
 
   if (XGetWMName (dpy, w, &text_prop) == 0)
     {
-      PRINT_ERROR (("XGetWMName failed\n"));
+      PRINT_DEBUG (("XGetWMName failed\n"));
       return NULL;
     }
 
